plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -278,5 +289,11 @@ def ProxyShapeFilePathRefresh(filePathAttr):
                 if res == kYes:
                     debugMessage('  User confirmed reload action, calling UsdStage.Reload()')
                     proxyStage.Reload()
+        
+        # Refresh the system lock status of the stage and its sublayers
+        stageFilePath = cmds.getAttr(filePathAttr)
+        fullStageName = GetFullStageNameFromProxyShapeAttr(filePathAttr)
+        cmds.mayaUsdLayerEditor(stageFilePath, edit=True, refreshSystemLock=(fullStageName, True))
```

Comment:
I guess the stageFilePath is really the root layer name we want to refresh?

Reply:
Yes, it's the root layer path/